tests/src/test/scala/ch/renku/acceptancetests/workflows/FlightsTutorial.scala

```diff
@@ -0,0 +1,79 @@
+package ch.renku.acceptancetests.workflows
+
+import ch.renku.acceptancetests.pages.JupyterLabPage
+import ch.renku.acceptancetests.tooling.HandsOnSpecData
+import ch.renku.acceptancetests.tooling.AcceptanceSpec
+
+import scala.concurrent.duration._
+import scala.language.postfixOps
+
+trait FlightsTutorial {
+  self: AcceptanceSpec with HandsOnSpecData =>
+
+  def followFlightsTutorial(jupyterLabPage: JupyterLabPage): Unit = {
+    import jupyterLabPage.terminal
+
+    // Create or import the dataset
+    val createDataSet = false
+    if (createDataSet) {
+      And("creates a dataset")
+      terminal %> "renku dataset create flights" sleep (2 seconds)
+      And("adds data to the dataset")
+      terminal %> "renku dataset add flights https://renkulab.io/gitlab/cramakri/renku-tutorial-flights-data-raw/raw/master/data/v1/2019-01-flights.csv.zip" sleep (13 seconds)
+    } else {
+      And("imports a dataset from the dataverse")
+      terminal %> "yes 2>/dev/null | renku dataset import 10.7910/DVN/WTZS4K" sleep (15 seconds)
```

Comment:
This is actually misspelled if the system keyboard layout is not `en_US`. It's not a big deal, not sure how difficult it may be to fix this